TESTS/configs/baremetal.json

```diff
@@ -37,6 +37,24 @@
     "target_overrides": {
         "*": {
             "mbed-trace.fea-ipv6": false
+        },
+        "K64F": {
+            "target.default_lib": "small"
+        },
+        "K66F": {
+            "target.default_lib": "small"
+        },
+        "NUCLEO_F303RE": {
+            "target.default_lib": "small"
+        },
+        "NUCLEO_F411RE": {
+            "target.default_lib": "small"
```

Comment:
Why target.default_lib is not set to small for all targets,
instead of adding a line for each 1...?

Reply:
we only added target.default_lib with small for the golden targets with green tea test passes.